test/multiverse/suites/tilt/Envfile

```diff
@@ -10,10 +10,16 @@ TILT_VERSIONS = [
   ['1.4.1', 2.2, 2.7]
 ]
 
+def haml_version(tilt_version)
+  if tilt_version && (Gem::Version.new(tilt_version.match(/\d|\./)) < Gem::Version.new('2.0.0'))
```

Comment:
From looking at our possible values for `tilt_version`, which currently is always going to either be `'1.4.1'` or `nil`, I think performing `[0]` on the version string would be better for getting at the first digit than a regex which contains an "or" pipe:

```diff
- (Gem::Version.new(tilt_version.match(/\d|\./))
+ (Gem::Version.new(tilt_version[0])
```

Granted, if you feel our 15 year old project will be around for another 15 years, neither this `[0]` approach nor the original regex one will handle double digit major version based strings.

If you want to handle that, you can do this:

```diff
- (Gem::Version.new(tilt_version.match(/\d|\./))
+ (Gem::Version.new(tilt_version.match(/^\d+/))
```